Game/Scripts/EnemyDroneScript.cpp

```diff
@@ -41,6 +42,9 @@ void EnemyDroneScript::Start()
 	attackScript = owner->GetComponent<DroneFastAttack>();
 	healthScript = owner->GetComponent<HealthSystem>();
 
+	seekTarget = seekScript->GetField<GameObject*>("Target")->getter();
```

Comment:
```suggestion
	seekTarget = seekScript->GetTarget();
```
`GetTarget()` does not exist right now, but you could create it real quick in the `SeekBehaviourScript`.

Reply:
Done